src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -48,6 +48,9 @@ public class TypicalPersons {
     public static final Person IDA = new PersonBuilder().withName("Ida Mueller")
             .withEmail("hans@example.com")
             .withPhone("84821311").build();
+    public static final Person JON_NOT_IN_TYPICAL_ADDRESS_BOOK = new PersonBuilder().withName("John Doe")
```

Comment:
```suggestion
    public static final Person JOHN_NOT_IN_TYPICAL_ADDRESS_BOOK = new PersonBuilder().withName("John Doe")
```
Perhaps change to above because the name is "John" rather than "Jon".

Reply:
Good catch!